ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -6529,6 +6529,149 @@ periodics:
         secretName: ci-pull-credentials
 - agent: kubernetes
   cluster: api.ci
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  interval: 24h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-env: openstack
+    job-release: "4.3"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-ocp-installer-e2e-openstack-ppc64le-4.3
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --kubeconfig=/etc/apici/kubeconfig
+      - --lease-server-password-file=/etc/boskos/password
+      - --lease-server-username=ci
+      - --lease-server=https://boskos-ci.svc.ci.openshift.org
+      - --secret-dir=/usr/local/e2e-openstack-cluster-profile
+      - --secret-dir=/usr/local/pull-secret
+      - --target=e2e-openstack
+      - --template=/usr/local/e2e-openstack
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.3
+      - name: BRANCH
+        value: "4.3"
+      - name: CLUSTER_TYPE
+        value: openstack-ppc64le
+      - name: CONFIG_SPEC
+        value: |
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tag_specification:
+            cluster: https://api.ci.openshift.org
+            name: "$(BRANCH)"
+            namespace: ocp
+          tests:
+          - as: e2e-$(CLUSTER_TYPE)
+            commands: TEST_SUITE=openshift/conformance/parallel run-tests
+            openshift_installer:
+              cluster_profile: "$(CLUSTER_TYPE)"
+      - name: JOB_NAME_SAFE
+        value: e2e-openstack
+      - name: KURYR_ENABLED
+        value: "false"
+      - name: TEST_COMMAND
+        value: |
+          curl -o excluded_tests https://gist.githubusercontent.com/mkumatag/ee4fdfa468f43f3cabffac945ccb843c/raw/a1ab152762f84ffbfd2d43e1f0aac58d002a45f3/ocp43_power_blacklist.txt
```

Comment:
This cannot be merged as is. In addition to the higher failure rates this will introduce, this undermines our ability to deliver a reproducible CI environment since we have no way of tracking the contents of that gist.

/hold

Reply:
It was always my plan to use a list in the code.  This was a temporary measure.